src/TestFramework/PhpUnit/Config/Builder/MutationConfigBuilder.php

```diff
@@ -65,12 +66,18 @@ class MutationConfigBuilder extends ConfigBuilder
      */
     private $dom;
 
-    public function __construct(string $tempDirectory, string $originalXmlConfigContent, XmlConfigurationHelper $xmlConfigurationHelper, string $projectDir)
+    /**
+     * @var JUnitTestCaseSorter
+     */
+    private $jUnitTestCaseSorter;
+
+    public function __construct(string $tempDirectory, string $originalXmlConfigContent, XmlConfigurationHelper $xmlConfigurationHelper, string $projectDir, JUnitTestCaseSorter $jUnitTestCaseSorter)
```

Comment:
line returns here? It's quite  along line

Reply:
done